.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "TLM/OptionsFramework"]
 	path = TLM/OptionsFramework
-	url = ../OptionsFramework.git
+	url = https://github.com/VictorPhilipp/OptionsFramework.git
 [submodule "TLM/CSUtil.CameraControl"]
 	path = TLM/CSUtil.CameraControl
-	url = ../CSUtil.CameraControl.git
+	url = https://github.com/VictorPhilipp/CSUtil.CameraControl.git
+[submodule "TLM/TLM/RedirectionFramework"]
+	path = TLM/TLM/RedirectionFramework
+	url = https://github.com/VictorPhilipp/RedirectionFramework.git
```

Comment:
@krzychu124 would need to fork this repo

Reply:
Done 😉